std/uni.d

```diff
@@ -4086,9 +4086,8 @@ public:
     */
     void putValue(Key key, Value v)
     {
-        import std.conv : text;
         auto idx = getIndex(key);
-        enforce(idx >= curIndex, text(errMsg, " ", idx));
+        enforce(idx >= curIndex, errMsg);
```

Comment:
Can't we do `errMsg ~ " " ~ idx`?
It's `lazy` after all :)

Reply:
idx is a integer.